### DISCUSSION TITLE
WORKING SESSION 28 DEC @ 11:30 EST / 17:30 CET

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1927] [General]
Where: [via Matrix](https://matrix.to/#/#project-mesa:matrix.org)

This is a continuation of this session: https://github.com/projectmesa/mesa/discussions/1918 

> Main focuses are: MEPs, Experimental Features, and auto deploy are all tied together.

Led by @EwoutH and @jackiekazil

**Comment by EwoutH** [comment]
Maybe we can dedicate a  part of the meeting to the 2.2.0 release. All features should be in (see https://github.com/projectmesa/mesa/pull/1917#issuecomment-1870625103), so I think the release process can start.

Maybe @tpike3 and @Corvince could join (for a portion) of it?

**Reply by jackiekazil** [reply]
I think that might be packing too much into one meeting. If we want to restructure for that -- we should optimized schedules for that. Open to adjustments if we want to change from a process meeting to release meeting... not sure others are ready to jump in though.

**Reply by EwoutH** [reply]
Let’s keep the original agenda in that case
